index.bs

```diff
@@ -261,39 +259,78 @@ vrDisplay.requestPresent([{ source: canvas }]).then(function() {
 
 ## VRLayer ## {#interface-vrlayer}
 
-The {{VRLayer}} interface is provided to a {{VRDisplay}} and presented in the HMD.
+The {{VRLayer}} is the base interface for the various layer types that can be
+provided to a {{requestPresent()}} call to be presented in the HMD.
 
-<pre class="idl" id="vrlayer-dictionary">
+<pre class="idl" id="vrlayer-interface">
+interface VRLayer {};
```

Comment:
One thing I want to call out that I like about this proposal is that it makes feature detection of supported layers really easy once we start defining new ones. It becomes simply:

```
if (EMEVideoVRLayer) {
  // Yay! I can use this layer type!
}
```